### PR TITLE
Run edgecore as system service

### PR DESCRIPTION
**Pull request by dingyin**
Co-authored-by: Yin Ding <dingyin@gmail.com>
Co-authored-by: Rio Zhu <32083634+zzxgzgz@users.noreply.github.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
/kind feature

**What this PR does / why we need it**:
The PR is to set edgecore running as a system process, so it will be automatically restarted when the system is rebooted.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: keadm/cmd/keadm/app/cmd/util/common.go [hunk 1]
```diff
@@ -486,3 +587,37 @@ func isKubeEdgeProcessRunning(proc string) (bool, error) {
...
+	if stdout != "" {
+		return true, nil
+	}
+
+	return false, nil
+}
+
+//	check if systemd exist
+func hasSystemd() bool {
+
```
Comment (kevin-wangzefeng) [inline keadm/cmd/keadm/app/cmd/util/common.go:609]:
```suggestion
```

File: keadm/cmd/keadm/app/cmd/util/common.go [hunk 2]
```diff
@@ -486,3 +587,37 @@ func isKubeEdgeProcessRunning(proc string) (bool, error) {
...
+	cmd := "file /sbin/init"
+
```
Comment (kevin-wangzefeng) [inline keadm/cmd/keadm/app/cmd/util/common.go:611]:
```suggestion
```

File: keadm/cmd/keadm/app/cmd/util/common.go [hunk 3]
```diff
@@ -44,10 +44,12 @@ const (
 	CentOSType     = "centos"
 
 	KubeEdgeDownloadURL          = "https://github.com/kubeedge/kubeedge/releases/download"
+	EdgecoreServiceFileURL       = "https://raw.githubusercontent.com/kubeedge/kubeedge/release-%s/build/tools/%s"
```
Comment (kevin-wangzefeng) [inline keadm/cmd/keadm/app/cmd/util/common.go:47]:
```suggestion
	EdgeCoreServiceFileURL       = "https://raw.githubusercontent.com/kubeedge/kubeedge/release-%s/build/tools/%s"
```

Comment (zzxgzgz) [reply keadm/cmd/keadm/app/cmd/util/common.go:47]: I find these two line looking the same, am I missing anything here?

Comment (dingyin) [reply keadm/cmd/keadm/app/cmd/util/common.go:47]: Bad diff? I can only see it is a new line added here.

File: keadm/cmd/keadm/app/cmd/util/common.go (hunk 2 above)
Comment (dingyin) [reply keadm/cmd/keadm/app/cmd/util/common.go:606]: fixed.

File: keadm/cmd/keadm/app/cmd/util/common.go (hunk 1 above)
Comment (dingyin) [reply keadm/cmd/keadm/app/cmd/util/common.go]: fixed.

File: keadm/cmd/keadm/app/cmd/util/common.go (hunk 3 above)
Comment (dingyin) [reply keadm/cmd/keadm/app/cmd/util/common.go]: fixed

**Comment by dingyin** [comment]
@kevin-wangzefeng @fisherxu Could we get it in?

**Pull request merged**